docroot/themes/custom/uids_base/scss/components/card.scss

```diff
@@ -26,3 +26,15 @@
     flex: 1 1 100%;
   }
 }
+
+.layout-builder-block.card {
```

Comment:
@joewhitsitt should we move this to `admin.scss`? This is just for the editor appearance, right?

Reply:
I thought I had to do this to ensure proper backgrounds and content colors regardless of the background. I can't remember if I did this based on the preview or the actual.

Reply:
I think that class only appears on the edit screen.

Reply:
I've noticed that issue while working on https://github.com/uiowa/uiowa/pull/2076.

Reply:
moved it. will test some more tomorrow when i work on webform support